project-reports/2023/2023-Q3-Hyperledger-AnonCreds.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: default
+title: 2023 Q3 Hyperledger AnonCreds
+parent: 2023
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+
+# 2023 Q3 Hyperledger AnonCreds
+
+Created by Stephen Curran and the Hyperledger AnonCreds Maintainers.
+
+## Implementation Repositories
+
+- [anoncreds-rs] - Rust implementation of AnonCreds
+- [anoncreds-clsignatures-rs] - Rust implementation of CL Signatures used in [anoncreds-rs]. Moved from Ursa and enhanced.
+
+[anoncreds-rs]: https://github.com/hyperledger/anoncreds-rs
+[anoncreds-clsignatures-rs]: https://github.com/hyperledger/anoncreds-clsignatures-rs
+
+### Specifications
+
+- [AnonCreds Specification] published from the repository [anoncreds-spec]
+- [AnonCreds Methods Registry] published from the repository [anoncreds-methods-registry]
+- [AnonCreds v2.0 Specification] published from the repository [anoncreds-spec-v2]
+
+[anoncreds-spec]: https://github.com/hyperledger/anoncreds-spec
+[AnonCreds Specification]: https://hyperledger.github.io/anoncreds-spec/
+[anoncreds-methods-registry]: https://github.com/hyperledger/anoncreds-methods-registry
+[AnonCreds Methods Registry]: https://hyperledger.github.io/anoncreds-methods-registry
+[anoncreds-spec-v2]: https://github.com/hyperledger/anoncreds-spec-v2
+[AnonCreds v2.0 Specification]: https://hyperledger.github.io/anoncreds-spec-v2/
+
+### Web and Wiki Pages
+
+- [Hyperledger AnonCreds Website Home Page]
+- [Hyperledger AnonCreds Wiki Home Page]
+
+[Hyperledger AnonCreds Website Home Page]: https://www.hyperledger.org/use/anoncreds
+[Hyperledger AnonCreds Wiki Home Page]: https://wiki.hyperledger.org/display/ANONCREDS/
+
+## Project Health
+
+The project continues to make good progress, with the maintainers and
+contributors completing version 0.1.0 of the AnonCreds Rust implementation and
+that library being used in downstream applications.
```

Comment:
"downstream applications" sounds vague, could use a clarification on whether that's "applications built by adopters of AnonCreds", or "applications in the AnonCreds project", or "applications built by other teams in the foundation"?